src/OnboardingSPA/components/Drawer/DrawerPanel/DesignColors.js

```diff
@@ -547,7 +656,7 @@ const DesignColors = () => {
 	return (
 		<div className="theme-colors--drawer">
 			<h2>{ __( 'Color Palettes', 'wp-module-onboarding' ) }</h2>
-			{ colorPalettes && buildPalettes() }
+			{ colorPalettes && buildPredefinedPalette() }
```

Comment:
What's the difference between `predefined` and `default` palette?

Reply:
Nothing, Just changed that to distinguish between Custom and the one we we have already defined, Can change it to default.